api/model/form/country_field.go

```diff
@@ -0,0 +1,276 @@
+package form
+
+import "fmt"
+
+// CountryField stores a country location
+type CountryField string
+
+// Valid ensures that an appropriate county is selected
+func (f CountryField) Valid() (bool, error) {
+	s := string(f)
+	for _, country := range countryList {
+		if s == country {
+			return true, nil
+		}
+	}
+	return false, ErrFieldInvalid{fmt.Sprintf("`%v` is an invalid country", s)}
+}
+
+var countryList = [...]string{
```

Comment:
Yeah, we were planning either a flat file or from a database in the near future.